packages/fxa-auth-server/lib/metrics/context.js

```diff
@@ -118,13 +118,9 @@ module.exports = function (log, config) {
   async function get(request) {
     let token;
 
+    // FIXME: this is very inefficient, because it tries to query Redis for all
+    // metrics context.
     try {
-      const metadata = request.payload && request.payload.metricsContext;
-
-      if (metadata) {
-        return metadata;
-      }
```

Comment:
Why this change?  This reverses the precedence of the metrics context from the payload vs the stashed one; it's likely to break things.